webapp/apps/test_assets/test_reform.py

```diff
@@ -0,0 +1,111 @@
+reform_text = """// Title: 2016 Trump Campaign Tax Plan
+// Reform_File_Author: Matt Jensen
```

Comment:
@MattHJensen at the time I was thinking about expedience. I think I wanted to rely on modules rather than file locations, since the latter is sometimes finicky.

I can get it from TaxCalc too, just have to do some `os.path` code to get it from the module location and then parse it as JSON.